### DISCUSSION TITLE
Userbase Survey

### DISCUSSION THREAD

**Discussion by aspickard** [dasher-project/redash#49] [UX]
It would be great to collect some existing user input so that we may properly prioritize our features and design decisions moving forward. Here we will discuss metrics we should gather from our users in order to better format our survey.

**Comment by willwade** [comment]
I suggest some key aspects like this (https://dasher.acecentre.net/about/#who-uses-dasher-right-now-what-features-do-we-need-to-support)

- Language
- Platforms
- Input method
- Output formats
- Corpus size 
- Daily use/est hours

**Comment by aspickard** [comment]
Jen has captured a lot of UX feedback here: [Google Doc](https://docs.google.com/document/d/1lffHLr33jn-my0hxrkCxZRQ3pgYlfGp-ii04EtuUKSo/edit?resourcekey=0-C7RnWXhvxWN2yVlvZHVacQ)

To add to what you mentioned Will, it might be valuable to know:

- Estimated input speed in WPM or another similar metric. 
- Favorite use case(s)
- Least favorite use case(s)

**Reply by willwade** [reply]
Yeah. I'm not sure how a user would estimate wpm speed (an aside. Note that Steve S wanted this as a feature in v5 https://github.com/dasher-project/dasher/issues/19). Guess could ask them to write a sentence and time it?
